README.md

```diff
@@ -961,6 +961,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Dušan Tanasić](https://github.com/Duk4)
   - [Dywa VaraPrasad](https://github.com/dywa-varaprasad)
   - [Dário Prazeres](https://github.com/DarioPrazeres)
+  - [Damaris Muange](https://github.com/ndush)
```

Comment:
Please add your name at the right position to keep this list in the correct alphabetical order. (It should be line 840.)